CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 ## 0.76.0 / 2025-08-08
 
-[FEATURE] OAuth2 TLSConfig and ProxyConfig settings. #6735
-[ENHANCEMENT] Use service endpoint role for alerting. #6784
-[FEATURE] Support configuring OAuth2 in ScrapeConfig. #6814
-[FEATURE] Add scale subresource to Alertmanger resource. #6728
-[ENHANCEMENT] set the namespace if empty for alertmanagers to the current namespace in Prometheus CR. #6338
-[ENHANCEMENT] support TLSConfig TLSVersion setting. #6736
+[CHANGE] Change the `secrets` field of Prometheus and PrometheusAgent CRDs from `atomic` to `set`. This allows different managers to manage entries separately. #6762
+[FEATURE] Add TLS and Proxy settings to OAuth2 configuration for Prometheus and PrometheusAgent CRDs. #6735
+[FEATURE] Add support for OAuth2 in the ScrapeConfig CRD. #6814
+[FEATURE] Add scale subresource to the Alertmanger CRD. #6728
+[FEATURE] Add Scaleway service discovery to the ScrapeConfig CRD. #6711
+[ENHANCEMENT] Make the `namespace` field optional in the Alertmanager endpoints configuration of the Prometheus CRD. #6338
+[ENHANCEMENT] Add support to configure the TLS version for Prometheus, PrometheusAgent and Alertmanager CRDs. #6736
 [ENHANCEMENT] optimize `TLSConfig` to `addSafeTLSToYaml`. #6726
-[ENHANCEMENT] Add ServiceDiscoveryRole configuration for use of Endpoints vs EndpointSlice. #6672
-[ENHANCEMENT] Add secret label selector to filter secrets to watch. #6731
-[CHANGE] Add listType:=set to Prometheus secrets field. This allows for having a different manager for each secret. #6762
-[ENHANCEMENT] Add attachMetadata field to ScrapeClasses. #6756
-[FEATURE] Add ScaleWay service discovery to the ScrapeConfig CRD. #6711
+[ENHANCEMENT] Refactor Kubernetes SD of ScrapeConfig by new adding validations. #6678
```

Comment:
```suggestion
[ENHANCEMENT] Add API-level validations to Kubernetes SD in the ScrapeConfig CRD. #6678
```

Let's remove whatever is not interesting to the final user :) (they don't care about refactorings)